templates/gabare_life/includes/_presse.html.twig

```diff
@@ -1,39 +1,43 @@
-<section class="bgPresse pb-5 links" id="dans-la-presse">
+<section class="bgPresse pb-5 links container-fluid" id="dans-la-presse">
+
     <h2 class="text-center p-3 p-md-5">Presse</h2>
-    <div class="container-fluid">
-        <div class="row flex-wrap justify-content-around ml-md-5 mr-md-5">
-
-            <div class="col-5 col-sm-5 ">
-                {% for press in presse %}
-                    <div class="card article mb-5 mb-md-0">
-                        <div class="card-body">
-                            <img class="picture-article mb-4" src="{{ press.image }}" alt="#">
-                            <h4 class="mb-3">{{press.title}}</h4>
-                            <p class="mb-3">{{ press.resume }}</p>
-                            <div class="show-link">
-                                <a href="{{ press.link }}" target="_blank" class="font-weight-bold">lire l'article complet ici <i class="fas fa-arrow-right"></i></a>
-                            </div>
-                        </div>
-                        {% endfor %}
+
+    <div class="row">
+
+
+        <div class="card article col-md-6">
+            {% for press in presse %}
```

Comment:
presse est en français. Tu as un pb de mot anglais invariable là
je pense qu'il faut simplement que tu renomme tes variable en qq chose comme
for pressArticle in pressArticles